portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +27,17 @@ setInterval(() => {
   }
 }, SPEED);
 
+function start() {
+  loadComments();
+  createMap();
+}
+
+function createMap() {
+  const map = new google.maps.Map(
+    document.getElementById('map'), 
+    {center: {lat: 37.4220041, lng: -122.0862515}, zoom: 15});
```

Comment:
Can you use constant var or comments to explain what this place is?

Reply:
When you say constant var, do you mean make a descriptive constant i.e. MTV_LAT to hold the latitude and longitude values?

Reply:
Yes, self documenting var name would be ideal.

Reply:
Ok will do, thank you!